Stripe/STPAddCardViewController.m

```diff
@@ -91,7 +90,7 @@ - (void)commonInitWithConfiguration:(STPPaymentConfiguration *)configuration {
     _configuration = configuration;
     _shippingAddress = nil;
     _hasUsedShippingAddress = NO;
-    _apiClient = [[STPAPIClient alloc] initWithConfiguration:configuration];
+    _apiClient = [STPAPIClient sharedClient];
```

Comment:
hmm, these changes are the most concerning to me. If `stripeAccount` (and other STPAPIClient configurations) is still a property on `STPPaymentConfiguration` this should probably respect those. Is the plan to deprecate on `STPPaymentConfiguration`?

Reply:
Yes, plan is to kill `PaymentConfiguration.{stripeAccount, publishableKey}` in a follow-up PR

Reply:
mind putting up that patch in parallel with this one so I can review both together?